inputfiles/overridingTypes.jsonc

```
@@ -2989,7 +3101,7 @@
                                     "param": [
                                         {
                                             "name": "callback",
-                                            "overrideType": "EventListenerOrEventListenerObject"
+                                            "overrideType": "EventListenerOrEventListenerObject<this>"
```

I would have expected we need:
```suggestion
                                            "overrideType": "EventListenerOrEventListenerObject<typeof this>"
```
But both works (in my tests in my project).